portal/src/main/webapp/networkProxy.jsp

```diff
@@ -0,0 +1,2 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<c:import url="https://rawgit.com/alisman/cbioportal-frontend/netFrame/src/pages/resultsView/network/network.htm" />
```

Comment:
maybe set this to specific commit after merging https://github.com/cBioPortal/cbioportal-frontend/pull/1290/files ?

Reply:
will do